apps/clapi/server/endpoints/academic/licence.js

```diff
@@ -146,7 +146,7 @@ CLapi.internals.academic.licence = function(url,resolve,content,start,end,refres
     })(resolved);
     if (info.retrievable) {
       for ( var k in info ) licence[k] = info[k];
-      if (refresh) {
+      if (exists && refresh) {
```

Comment:
After I enabled the academic licence checker to run, it failed on the first DOI I gave it with an error here. Seems straightforward, if no existing result found, don't try to update an existing result - make a new one.